java-frontend/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -70,7 +69,7 @@ public ProgramPoint(CFG.Block block, int i) {
 
     @Override
     public int hashCode() {
-      if(hashcode == 0) {
+      if (hashcode == 0) {
```

Comment:
Thanks for the parameter, useful indeed. But as discussed a bit ouside of the point : this, ideally,  should not be part of this commit.